layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,137 @@
+      <li>Delrawn Small</li>
+      <li>Sandra Bland</li>
+      <li>Botham Jean</li>
+      <li>Trayvon Martin</li>
+      <li>Jamar Clark</li>
+      <li>Laquan McDonald</li>
+      <li>Antwon Rose II</li>
+      <li>Keith Childress</li>
+      <li>Walter Scott</li>
+      <li>Alton Sterling</li>
+      <li>Stephon Clark</li>
+    </ul>
+    <p><a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="">far</a>, <a href="">too</a> <a href="">many</a> <a href="">more</a>.</p>
+  </section>
+
+  <section>
+    <p>Where you can donate to with your wallet:</p>
+    <ul>
+      <li>Donate to the <a href="https://www.gofundme.com/f/georgefloyd">Official George Floyd Memorial Fund</a>.</li>
+      <li>Donate to the <a href="https://www.gofundme.com/f/i-run-with-maud">I Run With Maud</a> fundraiser for Ahmaud Arbery.</li>
+      <li>Debate to the <a href="https://www.gofundme.com/f/justice-for-mike-ramos">Justice for Mike Ramos</a> fundraiser.</li>
+      <li>Find and donate to bail fund via the <a href="https://www.communityjusticeexchange.org/nbfn-directory">National Bail Fund Network</a>.</li>
+      <li><a href="https://www.facebook.com/reclaimtheblock/">Reclaim the Block</a> has published a <a href="https://docs.google.com/document/d/1yLWGTQIe3967hdc9RSxBq5s6KKZHe-3_mWp5oemd7OA/preview?pru=AAABcpUiX3k*Y6Q4I6UBtkH3lLz9GVLg0A">Google Doc of organizations to donate to.</a></li>
+    </ul>
+    <p>We will be updating this list periodically.</p>
+  </section>
+
+  <section id="our-support">
+    <p>In Node.js, we're going to be working on supporting Black members of the community. If we can help - by using our collective platforms to amplify your words, by helping you in your career or in contributing to open-source, or in any other way - we are here for you. Each one of us in the list below has put out a message (linked) asserting our support and willingness to help - feel free to ping us in these threads, via open DMs on the linked accounts, or by sending us an email. We are here for you.</p>
+
+    <section id="we-are-here-for-you">Tierney Cyren (<a href='https://twitter.com/bitandbang/status/1267893948149911556?s=20'>message of support</a>, <a href='https://twitter.com/bitandbang'>twitter</a>)</section>
+    
+    <section id="emails">Additionally, please feel welcome to reach out to the <a href="https://github.com/nodejs/node#tsc-technical-steering-committee">Node.js TSC members</a> and <a href="https://github.com/nodejs/community-committee#community-committee-members">Node.js Community Committee members</a> via our email addresses listed in the links, respectively.</section>
+  </section>
+
+  <section id="temporary">
+    If you're looking for docs, they're available in <a href="https://github.com/nodejs/node/tree/master/doc">nodejs/node</a>. If you're looking for the most recent security release, please see the temporary <a href="">security release landing page</a> in lieu of the downloads page and blog post.
```

Comment:
There is currently no `security landing page` what do we want to do with this.

The deep links for download and blog and whatnot still work, so might want to just link to the post

`/blog/vulnerability/june-2020-security-releases/`

Reply:
```suggestion
    If you're looking for docs, they're available in <a href="https://github.com/nodejs/node/tree/master/doc">nodejs/node</a>. If you're looking for the most recent security release, please see the temporary <a href="/blog/vulnerability/june-2020-security-releases/">security release landing page</a> in lieu of the downloads page and blog post.
```